Userland/Libraries/LibWeb/DOM/Document.cpp

```diff
@@ -371,6 +371,22 @@ Document::Document(JS::Realm& realm, const URL::URL& url, TemporaryDocumentForFr
         .has_legacy_override_built_ins_interface_extended_attribute = true,
     };
 
+    m_cursor_blink_timer = Core::Timer::create_repeating(500, [this] {
+        if (!m_cursor_position || !navigable() || !navigable()->is_focused())
```

Comment:
Can we avoid the double `navigable()` here? That function keeps showing up in profiles :(

Reply:
oof TIL that `navigable()` is not O(1). Done, and moved the other cheaper early returns before it